ads/microad.md

```diff
@@ -22,14 +22,14 @@ limitations under the License.
 <amp-ad width="320" height="50"
 	type="microad"
     data-spot="b4bf837c5ddd69758d5ac924d04cf502"
-    data-url=""
-    data-referrer="">
+    data-url="${COMPASS_EXT_URL}"
```

Comment:
The code above is just an example.  If used in your markup the result will not be specific to your account.  You want to use the MicroAd Compass Interface, and copy the code found inside the box labeled "Ad Tag".